src/OpenOrchestra/Tests/Functional/ModelBundle/Repository/ContentRepositoryTest.php

```diff
@@ -440,6 +442,36 @@ public function provideFindByAuthorAndsiteId()
     }
 
     /**
+     * @param string       $user
+     * @param string       $siteId
+     * @param boolean|null $published
+     * @param int          $limit
+     * @param array|null   $sort
+     * @param int          $count
+     *
+     * @dataProvider provideFindByReportAndSiteId
+     */
+    public function testFindByReportAndSiteId($user, $siteId, $published, $limit, $sort, $count)
```

Comment:
@alavieille 
testFindByHistoryAndSiteId is in ContentRepositoryInterface but not in ReadContentRepositoryInterface. No test in front.